docs/src/routes/docs/components/autocomplete.svx

```diff
@@ -0,0 +1,104 @@
+---
+name: Autocomplete
+relatedComponents:
+  - name: AutocompleteField
+    link: ./docs/components/autocomplete-field
+  - name: AutocompleteOption
+    link: ./docs/components/autocomplete-option
+---
+
+<script>
+  import { Autocomplete } from 'attractions';
+  import { ChevronDownIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+  function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+</script>
+
+
+# Autocomplete
+
+A text field suggesting options to select.
+
+Builds upon the [==`AutocompleteField`==](./docs/components/autocomplete-field), prefer that instead if you need more customization freedom.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase">
+    <Autocomplete {getOptions} />
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  async function* getOptions(text) {
+    await sleep(1000);
+    yield [
+      { name: text, details: 'Optional' },
+      { name: `it highlights the match: ${text}` },
+    ];
+  }
+</script>
+
+<Autocomplete {getOptions} />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| ==**`getOptions`**==&nbsp;<span class="required">*</span> | `async function*` | An async generator of suggestions. <br /> Receives the input from the text field and is expected to yield promises that resolve to arrays of objects `{ name, details }`. |
+| **`selection`** | `Array` | The current selection as an array of objects `{ name, details }`. <br /> Can be used to set the selection programmatically. |
+| **`minSearchLength`** | `number` | The minimum amount of characters to call ==`getOptions`==. <br /> 3 by default. |
+| **`maxOptions`** | `number` | The maximum amount of options than can be selected. Set to `null` (the default) for no restrictions. |
+| **`searchQuery`** | `number` | The current value of the text field. Can be used to control the query programmatically. |
+| **`optionComponent`** | Svelte component | The component used to render a suggestion in the list. <br /> Receives two props: the `option` object (`{ name, details }`) and the current `query`. <br /> Expected to dispatch `click` events on selection. <br /> Defaults to [==`AutocompleteOption`==](./docs/components/autocomplete-option). |
+| **`...`** | `any` | Everything else will be passed to the underlying `<AutocompleteField>` component. |
```

Comment:
Isn't this note enough instead of listing the props above? Maintaining the same list of props in 2 places is not something pleasant

Reply:
Jumping around doc pages when you're trying to figure out a component isn't pleasant either

Reply:
Then how about refactoring this data to a separate file and importing it in both?